src/platforms/common/swdptap.c

```diff
@@ -36,7 +36,9 @@ static void swdptap_turnaround(uint8_t dir)
 	if(dir == olddir) return;
 	olddir = dir;
 
+#if 0
```

Comment:
Maybe use a `#ifdef` with a named macro `DEBUG_SWD_BITS` or similar.  If someone wants one of these, they will probably want them all.